pisa/core/distribution_maker.py

```diff
@@ -203,7 +202,8 @@ def get_outputs(self, return_sum=False, sum_map_name='total',
         outputs = [pipeline.get_outputs(**kwargs) for pipeline in self] # pylint: disable=redefined-outer-name
         if return_sum:
             if len(outputs) > 1:
-                outputs = reduce(lambda x, y: sum(x) + sum(y), outputs)
+                outputs = sum([sum(x) for x in outputs])
```

Comment:
No need to create a list and then run `sum` over the list;, `sum` works directly with an iteration:
```python
sum(sum(x) for x in outputs)
```